src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/utils/is-caterpillar-category.ts

```diff
@@ -0,0 +1,10 @@
+import type { Feature as FeatuesGeo } from 'geojson'
```

Comment:
I think it is good to be consistent. Else you will be easily confused since Feature is a custom type within the application and where they are used together this one is aliased.